pvlib/temperature.py

```diff
@@ -443,6 +444,50 @@ def faiman(poa_global, temp_air, wind_speed=1.0, u0=25.0, u1=6.84):
     return temp_air + temp_difference
 
 
+def ross(poa_global, temp_air, noct):
+    r'''
+    Calculate cell temperature using the Ross model.
+
+    The Ross model [1]_ assumes that cell temperature is proportional to the
+    plane of array irradiance, and ignores the effects of wind.
```

Comment:
Strictly speaking, it is the difference between ambient and cell temperature that is proportional to irradiance, not the cell temperature itself.